.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node: [6, 8, 12]
+        node: [10, 12]
```

Comment:
Did you drop **node 8** here too?

Reply:
Of course. It's also EOL. I used what we use in master which is the current LTS

Reply:
Ok, I adjusted the PR title